src/midifallback.c

```diff
@@ -17,6 +17,7 @@
 
 
 #include "doomtype.h"
+#include "m_io.h"
```

Comment:
Just use `#include <stdio.h>` instead of `m_io.h`.

Reply:
Fixed by https://github.com/fabiangreffrath/woof/pull/1053/commits/2c327067eac4dc3b5860002e9cd4a80d0b0ef6aa